modules/ppcp-api-client/src/Endpoint/PayUponInvoiceOrderEndpoint.php

```diff
@@ -191,10 +192,7 @@ public function order_payment_instructions( string $id ): array {
 			throw new PayPalApiException( $json, $status_code );
 		}
 
-		return array(
-			$json->payment_source->pay_upon_invoice->payment_reference,
-			$json->payment_source->pay_upon_invoice->deposit_bank_details,
-		);
+		return $json;
```

Comment:
probably can create an order like this?

https://github.com/woocommerce/woocommerce-paypal-payments/blob/7aca62a1a30fbcef72e5e8e7edb96b5fcad45c1f/modules/ppcp-api-client/src/Endpoint/OrderEndpoint.php#L294